docker/r/Dockerfile

```diff
@@ -31,8 +31,9 @@ RUN echo "PATH=${PATH}" >> /usr/local/lib/R/etc/Renviron && \
 ENV LD_LIBRARY_PATH /usr/local/lib/R/lib
 
 # pin the version of RStudio
-ENV RSTUDIO_VERSION 1.3.959
-RUN /rocker_scripts/install_rstudio.sh
+# ENV RSTUDIO_VERSION 2021.09.2+382
+# RUN /rocker_scripts/install_rstudio.sh
+# This version of rocker should already have RStudio installed!
```

Comment:
it's probably a good idea to pin the version of RStudio to avoid surprises anyway, even if it's just to the one that this rocker image is shipping with atm.

Reply:
yep. btw after 1.4.XYZ they moved to calendar versioning: https://www.rstudio.com/products/rstudio/release-notes/